packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -234,6 +234,39 @@ class IdentityDeletionProcess {
     }
 }
 
+class DeciderModule {
+    public doesNotHaveAutomationConfig() {
+        return new ApplicationError("error.runtime.decide.doesNotHaveAutomationConfig", "The Request can't be decided automatically, since no automationConfig was provided.");
+    }
```

Comment:
I don't understand why this is an error. Why can't we then simply let the user decide?

Reply:
@jkoenig134  Actually, I use all "DeciderModule RuntimeErrors" only within the DeciderModule to return error Results I can handle there and they are never returned to a user. Probably, I shouldn't list them here in the RuntimeError file and just define them within the DeciderModule, should I?

Reply:
if it never get's shown to the user please don't use error codes at all.

Reply:
I use Result.fail() which expects an ApplicationError and there a code is required.

Reply:
that's not good. you should just throw then

Reply:
But we shouldn't throw inside the DeciderModule. It must continue running and e.g. set the Request to ManualDecisionRequired. 🤔

Reply:
try..catch?

Reply:
wait for changing anything, I'll look deeper into it and come back to you.

Reply:
I think I would solve this buy just returning an object in this form: `{ wasDecided: true; requestId: string } | { wasDecided: false }` we should not use Result here as it forces us to introduce errors.

Reply:
Actually, we don't seem to need the requestId.